notebooks/lesson07_classification.ipynb

```diff
@@ -0,0 +1,3932 @@
+{
```

Comment:
<p>This had no influence on the performance, right? You could mention that you didn't categorise tenure for performance improvement but for interpretability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='148'/>

Reply:
<p>thanks!</p>